### ISSUE TITLE
ONNX parser reshape operator support

### ISSUE THREAD

**Issue by xinchenhawaii** [ARM-software/armnn#397]
I define forward function of MobilenetV2 in imagenet
def _forward_impl(self, x):
        # This exists since TorchScript doesn't support inheritance, so the superclass method
        # (this one) needs to have a name other than `forward` that can be accessed in a subclass
        x = self.features(x)
        # Cannot use "squeeze" as batch-size can be 1 => must use reshape with x.shape[0]
        x = nn.functional.adaptive_avg_pool2d(x, 1).reshape(x.shape[0], -1)
        x = self.classifier(x)
        return x
and convert to onnx model. However, I got the following errors:
Loading graph...
terminate called after throwing an instance of 'armnn::ParseException'
  what():  Unsupported operation Shape for node 'Shape_150'  at function LoadGraph [/home/pi/armnn-onnx/armnn/src/armnnOnnxParser/OnnxParser.cpp:665]
Aborted

**Comment by Colm-in-Arm** [comment]
Hi @xinchenhawaii 

I won't pretend I can follow what you're trying to achieve. However, if you're trying to load an Onnx "Shape" operator into our Onnx parser then I'm afraid we don't support it. We do support the "Reshape" operator. If you can find some way to avoid the call to shape then I think it should work.

Colm.

**Comment by matthewsloyanARM** [comment]
Hi @xinchenhawaii,

Thank you for reaching out! Based on the updated `README.md` found [here](https://github.com/ARM-software/armnn/tree/master#how-to-get-involved) under the "How to get involved" section we will be closing GitHub issues labeled with "Feature Request". Instead, we will be tracking the issue and adding the "Help wanted" label. All the issues with this label can be found [here](https://github.com/ARM-software/armnn/issues?q=is%3Aissue+label%3A%22Help+wanted%22+). We would much appreciate your help to add these features. If you would like to add this or any feature to Arm NN feel free to reopen this ticket and let us know you are working on it with a quick comment. We will then track the updates and help you get your additions into our codebase.

Below you will find our contributors guide which will show you how you can submit a patch to us. If you have any questions feel free to ask here or by creating a new Issue, thank you!

https://www.mlplatform.org/contributing

Kind regards,

Matthew
